rfcs/ambassador-program/index.md

```diff
@@ -0,0 +1,83 @@
+## GraphQL Ambassador Program
+
+The following is the beginnings of a document outlining the goals, outcomes and tasks of an Ambassador program. 
+
+### Alternative Names
+
+* Champion - already used on the technical side - Champions of spec edits
+* Evangelist - usually out of favour nowadays
+* Advocate - suggests a salaried role?
+* Leader, Star, Hero, Captain, Expert - “Star” we have already used in a recognition role
+* GraphQL Fellowship 
+
+> [!NOTE]
+> “Ambassador” seems to be readily accepted, I don't think we need to discuss the term any further.   
+
+### Ambassador Initiative Goals
+
+The overarching goal is to increase GraphQL adoption. The Ambassadors program aims to accomplish this through:
+
+* Brand advocacy - developers organically vouch for and promote GraphQL
+* Social proof - named individuals with high visibility (and a proportion from known corporate brands) as Ambassadors
+* Increased GraphQL content online and off
+* Community building - increase the number of “known faces” within GraphQL, increased approachability (addressing the concern of “lowering the barrier to join an inclusive GraphQL community” raised by the Community WG)  
+
+How can these goals be measured? 
+
+* Engagement metrics - social, chat channels, search, downloads, page views…
+* New community members (across social media comments, Discord users, local gathering attendees, working group members…) 
+* Volume of talks at conferences
+* Volume of articles written
+
+### Ambassador Tasks
+
+As a GraphQL Ambassador, you will be an important touchstone for people interested in GraphQL to learn more about it in a friendly and constructive way. 
+
+One or more of the following:
+
+* Conference & meetup talks on GraphQL
+* Local gathering organization
+* Written GraphQL content
+* Video / audio GraphQL content
+
+### Foundation Tasks
+
+* Grant available for travel expenses for approved conference & meetup talks
```

Comment:
Proposal to add "on a case by case basis". Some conferences out there are for-profit (even if it's becoming harder to turn a profit) and should participate in the fair retribution of their speakers.

```suggestion
* Grant available on a case by case basis for travel expenses for approved conference & meetup talks
```

Reply:
Much better wording, thanks both!